video/cloud-client/analyze/analyze.py

```diff
@@ -69,28 +58,24 @@ def analyze_explicit_content(path):
 
 def analyze_faces(path):
     """ Detects faces given a GCS path. """
-    video_client = videointelligence_v1beta2.VideoIntelligenceServiceClient()
-    features = [enums.Feature.FACE_DETECTION]
+    video_client = videointelligence.VideoIntelligenceServiceClient()
+    features = [videointelligence.enums.Feature.FACE_DETECTION]
 
-    config = types.FaceDetectionConfig(include_bounding_boxes=True)
-    context = types.VideoContext(face_detection_config=config)
+    config = videointelligence.types.FaceDetectionConfig(
+        include_bounding_boxes=True)
+    context = videointelligence.types.VideoContext(
+        face_detection_config=config)
 
     operation = video_client.annotate_video(
-        path, features, video_context=context)
+        path, features=features, video_context=context)
     print('\nProcessing video for face annotations:')
 
-    while not operation.done():
-        sys.stdout.write('.')
-        sys.stdout.flush()
-        time.sleep(15)
-
+    result = operation.result(timeout=600)
     print('\nFinished processing.')
 
     # first result is retrieved because a single video was processed
-    face_annotations = (operation.result().annotation_results[0].
-                        face_annotations)
-
-    for face_id, face in enumerate(face_annotations):
+    for face_id, face in enumerate(result.annotation_results[0].
+                                   face_annotations):
```

Comment:
Please never use this style of continuation. Either do this:

```python
for face_id, face in enumerate(
        result.annotation_results[0].face_annotations):
    print('Face {}'.format(face_id))
    print('Thumbnail size: {}'.format(len(face.thumbnail)))
```

or do this (preferred in this case):


```python
faces = result.annotation_results[0].face_annotations
for face_id, face in enumerate(faces):
    print('Face {}'.format(face_id))
    print('Thumbnail size: {}'.format(len(face.thumbnail)))
```

Reply:
done.